### PR TITLE
updated logic to validate the time and check if it is before the star…

### PR DESCRIPTION
**Pull request by MbfloydIR**
…t time, so it can be adjusted

## Summary
https://tsu.atlassian.net/browse/IR-5096
## References
closes #_insert number here_

### CODE REVIEW

File: packages/engine/src/scene/components/MediaComponent.ts [hunk 1]
```diff
@@ -192,6 +192,15 @@ export function MediaReactor() {
 
   if (!isClient) return null
 
+  function validateTime() {
+    const mediaElementState = getMutableComponent(entity, MediaElementComponent)
```
Comment (SamMazerIR) [inline packages/engine/src/scene/components/MediaComponent.ts]: i'd rename mediaElementState to mediaElementComponent here

File: packages/engine/src/scene/components/MediaComponent.ts [hunk 2]
```diff
@@ -192,6 +192,15 @@ export function MediaReactor() {
...
+    const element = mediaElementState.element.value as HTMLMediaElement
+    console.log('element.currentTime = ' + element.currentTime)
```
Comment (SamMazerIR) [inline packages/engine/src/scene/components/MediaComponent.ts]: remove console.log

Comment (MbfloydIR) [reply packages/engine/src/scene/components/MediaComponent.ts]: implemented

File: packages/engine/src/scene/components/MediaComponent.ts (hunk 1 above)
Comment (MbfloydIR) [reply packages/engine/src/scene/components/MediaComponent.ts]: implemented

**Review by SamMazerIR — APPROVED** [review]

**Pull request merged**